### DISCUSSION TITLE
Cell count real vs Cell count settings

### DISCUSSION THREAD

**Discussion by Sleeper85** [txubelaxu/esphome-jk-bms#40] [General]
Several users have reported to me an error in calculating the actual number of cells.

![image](https://github.com/user-attachments/assets/de09c71c-b1e1-42b0-a6bf-c5b16e39ad63)

**Comment by smpltechno** [comment]
That screenshot is from a 16S BMS-
![Screenshot_650](https://github.com/user-attachments/assets/30017b25-add5-446d-a141-5e55e7a9392f)

![Screenshot_649](https://github.com/user-attachments/assets/d6a8e593-f964-46ad-a694-7ab894949510)

**Comment by txubelaxu** [comment]
solved now (I think) 😉

**Reply by Sleeper85** [reply]
It would be nice if you opened the `issue` section of your github.

It's better for managing problems than in a discussion.

![image](https://github.com/user-attachments/assets/69a220f8-5a8b-4a7d-ad3b-f9b1fa3104ca)

For info I integrated your component for users of the [PVbrain2 PCB](https://github.com/SeByDocKy/pvbrain2).

Thank you for your work.
